docs/local-kuar.md

```diff
@@ -0,0 +1,43 @@
+# Local playground for testing and development
+
+Make sure you have all the tools mentioned in [this section](./local.md#environment-prerequisites) installed.
+
+First spin up two local k3s clusters:
+
+```sh
+make deploy-full-local-setup
+```
+
+Again, you can verify that everything is up and running by following [the steps here](./local.md#verify-installation).
+
+## Deploy the Kuar app
+
+```sh
+make deploy-kuar-app
+```
+
+This task will deploy a simple web application called Kuar into both clusters and exposes it. It also patches the installed nginx controller to serve the app even if the `Host` header is not provided in the request. This way you can access the application in your browser under http://localhost:80 for cluster 1 and http://localhost:81 for cluster 2.
+
+Make sure the app on http://localhost is responding, it may take a minute for the nginx ingress controller to restart with the correct parameters.
+
+## Kuar
+
+This application contains couple of handy features that can verify the k8gb functionality. It can resolve DNS names, simulate probe failures, can simulate a cpu intensive workload by generating RSA key pairs and also contains a simple api for queue (push & pop strings).
```

Comment:
```suggestion
This application contains a couple of handy features that can verify the k8gb functionality. It can resolve DNS names, simulate probe failures, can simulate a CPU-intensive workload by generating RSA key pairs, and also contains a simple API for the queue (push & pop strings).
```